client-oid4vc.json

```diff
@@ -110,6 +130,16 @@
         "userAttribute": "lastName",
         "supportedCredentialTypes": "identity_credential"
       }
+    },
+    {
+      "name": "iat-oid4vc-issued-at-time-claim-mapper-identity_credential",
+      "protocol": "oid4vc",
+      "protocolMapper": "oid4vc-issued-at-time-claim-mapper",
+      "config": {
+        "truncateToTimeUnit": "HOURS",
+        "valueSource": "COMPUTE",
+        "supportedCredentialTypes": "identity_credential"
+      }
```

Comment:
Unlike `test-credential`, an `nbf` mapper is missing for `identity_credential`. Is it intentional?

Reply:
It was intentional. I you need one there, just add one and send a pull request.